app/views/LicensorViewComponent.vue

```diff
@@ -53,15 +53,63 @@ div.licensor.container(v-else)
         td.border {{prepaid.endDate}}
         td.border {{prepaid.used}} / {{prepaid.maxRedeemers || 0}}
 
-    h3 Create API Client
+    h3 Create/Edit API Client
```

Comment:
Does this reflect existing settings?
If not, would it be much effort to show existing settings for an existing client in the edit scenario?

Reply:
It doesnt show existing settings yet, we can see them from show api client as of now. It would require some UI management effort to do this, so can it be done as a separate task.

Reply:
👍